trusona-sdk-http/src/main/java/com/trusona/sdk/http/client/v2/response/TrusonaficationResponse.java

```diff
@@ -77,11 +78,13 @@ public void setResource(String resource) {
   }
 
   public Date getExpiresAt() {
-    return expiresAt;
+    return expiresAt != null ? new Date(expiresAt.getTime()) : null;
```

Comment:
Was this a bug that was found? This feels like extra code for no reason. If so, what was the bug?

Reply:
`Date` objects are mutable - this prevents mutability on our date objects that we are holding

ideally, if i was to rewrite this, i would go with the long value

https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#ei-may-expose-internal-representation-by-returning-reference-to-mutable-object-ei-expose-rep